slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -253,34 +292,24 @@ void SLAMServiceImpl::CreateMap() {
             map_builder.map_builder_->GetTrajectoryBuilder(trajectory_id);
     }
 
-    std::vector<std::string> file_list =
-        viam::io::ListFilesInDirectory(path_to_data);
-    if (file_list.size() == 0) {
-        throw std::runtime_error("no data in data directory");
-    }
-    {
-        std::lock_guard<std::mutex> lk(map_builder_mutex);
-        map_builder.SetStartTime(file_list[0]);
-    }
-
-    if (starting_scan_number < 0 ||
-        starting_scan_number >= int(file_list.size())) {
-        throw std::runtime_error("starting_scan_number is out of bounds: " +
-                                 std::to_string(starting_scan_number));
-    }
-
-    std::cout << "Beginning to add data....\n";
+    LOG(INFO) << "Beginning to add data...";
 
     std::ofstream myfile;
     myfile.open("log.txt");
```

Comment:
only question is do we still want this? Not sure what it is/where its going

Reply:
@kkufieta was interested in keeping this for now.

Reply:
It's just helpful example code for the "Extract pose for grpc endpoint" ticket: https://viam.atlassian.net/browse/DATA-158 

With which it should be removed for sure.